app/metrics.yaml

```diff
@@ -18,28 +18,42 @@ events:
       startups, not just cold startup. Note: There is a short gap
       between the time application goes into background and the time
       android reports the application going into the background.
-      Note: This metric does not cover the following cases:
-      Case # 1 -> a). open a link(for example, gmail) with in-app
-      Browser (metric report custom_tab startup) b). press home button
-      c). open gmail again (which brings us back to in app browser).
-      Step c will not report startup metric. Case # 2 -> a). open fenix
-      b). press home button c). launch fenix through app switcher/recent
-      apps. step c will not report startup type.
+      Note: This metric does not record souce when app opened from
+      task switcher: open application -> press home button -> open
+      recent tasks -> choose fenix. In this case will report
+      [source = unknown, type = hot, has_saved_instance_state = false].
+      unknown is for tasks switcher or other unanticipated startup source.
```

Comment:
nit: -> "unknown is for startup sources where we can't pinpoint the cause. One UNKNOWN case is the app switcher where we don't know what variables to check to ensure this startup wasn't caused by something else"

Or something similar but shorter.

We should move this to the "source" description field rather than explain it in the top level.